spec/jobs/risc_delivery_job_spec.rb

```diff
@@ -96,6 +96,7 @@
 
           it 'logs an event' do
             expect { perform }.to_not raise_error
+            expect(NewRelic::Agent).not_to receive(:notice_error)
```

Comment:
So the code in this test never triggers a call to NewRelic (that happens as part of the [`GoodJob.on_thread_error`](https://github.com/18F/identity-idp/blob/a756b31c1b9f30c5c59b80afc44b76d8771553e7/config/application.rb#L110) that we set elsewhere)

So since this wouldn't catch a regression, I would remove it